src/Compilers/Core/Portable/CodeGen/ILBuilderEmit.cs

```diff
@@ -109,7 +109,8 @@ internal void EmitStackAllocBlockInitializer(ImmutableArray<byte> data, SyntaxNo
             }
             else
             {
-                var field = module.GetFieldForData(data, syntaxNode, diagnostics);
+                // alignment==1 as there's no special need for alignment (.pack) here.
+                var field = module.GetFieldForData(data, alignment: 1, syntaxNode, diagnostics);
```

Comment:
:memo: We have a single caller that comes into this branch (`emitInitBlock: false`) and it only calls this method when `elementType.SpecialType.SizeInBytes() == 1`. So `alignment: 1` is correct.
Similarly with `EmitArrayBlockFieldRef` below, we're fine with `alignment: 1` because it's only ever called when `specialElementType.SizeInBytes() == 1`.